### DISCUSSION TITLE
Is secure-electron-template active?

### DISCUSSION THREAD

**Discussion by SensitiveWebUser** [reZach/blog#12] [General]
Hello everyone,

Wanted to add a discussion to see what the state is for the [secure-electron-template](https://github.com/reZach/secure-electron-template) repo. Is it still active, or does it need to be archived? 

It’s a great template and would love to see it continue

Thanks

**Comment by reZach** [comment]
Thanks @SensitiveWebUser. The template itself is stable, but indeed you are correct it is not using the most current version of Electron and needs to be updated in order to be on the bleeding-edge of the framework.

**Reply by reZach** [reply]
I'm currently planning to work on a refresh with the latest framework, @SensitiveWebUser.

**Reply by SensitiveWebUser** [reply]
Oh cool! That would be great.
